server/eventlog.go

```diff
@@ -117,7 +145,9 @@ func contains(set [][]byte, value []byte) bool {
 	return false
 }
 
-func getVerifiedCosState(coscel cel.CEL) (*pb.AttestedCosState, error) {
+// getVerifiedCosState takes in CEL and a register type (can be PCR or RTMR), and returns the state
+// in the CEL. It will only include events using the correct trustingRegisterType.
+func getVerifiedCosState(coscel cel.CEL, trustingRegisterType uint8) (*pb.AttestedCosState, error) {
```

Comment:
registerType is fine. You can document that the caller should pass in the correct one.

Reply:
done